<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="back_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: red;
	border-radius: 20px;
	font-size: 15px;
	color: white;
}
QPushButton:hover {
	background-color: black;
}</string>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">color: red;
font-size: 55px;</string>
       </property>
       <property name="text">
        <string>Available Seats:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="details">
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font-size: 15px;</string>
     </property>
     <property name="text">
      <string>Movie:, Theatre:, Time:,</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>780</width>
        <height>431</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:  red;
padding: 20px;
border-radius: 20px;</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="book_button">
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(255, 0, 0);
	font-size: 20px;
}
QPushButton:hover {
	background-color: rgb(134,0, 0);
	border-radius: 10px;
}</string>
     </property>
     <property name="text">
      <string>Book</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
